src/navigations/HomeBottomTabs.js

```diff
@@ -193,7 +178,6 @@ function HomeBottomTabs({navigation}) {
 
   React.useEffect(() => {
     createChannel();
-    requestPermission();
     updateProfileAtomId();
 
     messaging().setBackgroundMessageHandler(async (remoteMessage) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `requestPermission()` function call from the `useEffect` hook means that the app will no longer request notification permissions when this component mounts. As mentioned above, this could potentially lead to issues if the user has not granted these permissions manually. Please ensure that there is a manual prompt for the user to grant these permissions elsewhere in the code.

```diff
-    requestPermission();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->